api/src/features/todo/todo_feature.py

```diff
@@ -57,8 +57,8 @@ def delete_todo_by_id(
 @router.get("", operation_id="get_all")
 def get_todo_all(
     user: User = Depends(auth_with_jwt), todo_repository: TodoRepositoryInterface = Depends(get_todo_repository)
-):
-    return get_todo_all_use_case(user_id=user.user_id, todo_repository=todo_repository)
+) -> list[GetTodoAllResponse]:
+    return get_todo_all_use_case(user_id=user.user_id, todo_repository=todo_repository)  # type: ignore
```

Comment:
Why the `# type: ignore` here?

Reply:
I could not figure it out, Mypy insists it is returning None, but the function has return type `list[GetTodoAllResponse]`.

So i basicly gave up